docs/source/user/basic/list.rst

```diff
@@ -0,0 +1,13 @@
+=======
+Listing
+=======
+
+Pet provides listing of:
+
```

Comment:
I tried based on already existing project and it doesn't work also editor error is raised:
```
| => pet init -i -t pet1 -n pet-project
pet1 - is project not template
/bin/sh: line 10: /usr/bin/nano: No such file or directory
/bin/sh: line 10: /usr/bin/nano: No such file or directory
```